indico_themes_canonical/static/css/_default/header.css

```diff
@@ -58,36 +103,45 @@ The style for the title text.
   line-height: 1.5rem;
 }
 
+.datePlace {
+  font-size: 1.306rem;
+}
+
 /* Overides large margins below custom registration fields */
 .ui.form.i-box-content .field:last-child {
   margin-bottom: 0;
 }
 
 /* Style for the announcement */
 .simpleTextAnnouncement {
-  background: none;
   border: none;
   color: var(--brand-white) !important;
+  background: var(--brand-background-primary);
```

Comment:
```suggestion
  background: transparent;
```

Reply:
The changes you made definitely work draw attention to the announcement, so if you are happy with how it looks, let's leave it as is.